lss_arm_moveit/config/joint_limits.yaml

```diff
@@ -11,35 +11,35 @@ joint_limits:
   lss_arm_joint_1:
     has_velocity_limits: true
     max_velocity: 6.283
-    has_acceleration_limits: false
-    max_acceleration: 0.0
+    has_acceleration_limits: true
```

Comment:
I know this stuff is not required. It's in a separate commit that I can easily remove. It just works better like this for me.